models/brac/function/get_hmis_data_function.sql

```diff
@@ -236,11 +236,11 @@ LEFT JOIN
 		meta.parent_uuid as area_uuid,
 		COUNT(person.uuid) FILTER(
 			WHERE extract (YEAR from age(now()::date, to_date(person.date_of_birth,'YYYY-MM-DD')))::int BETWEEN 12 AND 52  
-			AND person.sex ='female' AND (date_trunc('hour',person.reported)::DATE) <= date_trunc('hour',end_date)::DATE
+			AND person.sex ='female' AND person.reported::DATE <= (end_date)::DATE
```

Comment:
This function was originally named `get_hmis_data`. Could we maintain original naming semantics? Unsure if there are other cases of such

Reply:
It was originally `(date_trunc('hour',person.reported)::DATE) <= date_trunc('hour',end_date)::DATE` for both so I removed `(date_trunc('hour',...` for both because `::DATE` does the same thing, it truncates the hour and leaves only date.